forte/mrdsrg-spin-integrated/dsrg_mrpt2.h

```diff
@@ -175,6 +178,190 @@ class DSRG_MRPT2 : public MASTER_DSRG {
     /// Unitary matrix to block diagonal Fock
     ambit::BlockedTensor U_;
 
+
+    // NOTICE These are essential variables and functions for computing DSRG-MRPT2 gradient.
+    // Some variables may be redundant thus need further elimination
```

Comment:
Please do so. And also add comments.

Reply:
I want keep the variables unchanged for now until the frozen orbital approximation is done.

Reply:
All variables should end with `_` to signify that they are class variables and not local function variables.